library/psa_crypto_driver_wrappers.c

```diff
@@ -1332,7 +1332,7 @@ psa_status_t psa_driver_wrapper_hash_setup(
 #if defined(PSA_CRYPTO_DRIVER_CC3XX)
     status = cc3xx_hash_setup(&operation->ctx.cc3xx_driver_ctx, alg);
     if( status == PSA_SUCCESS )
-        operation->id = PSA_CRYPTO_CC3XX_DRIVER_ID;
+        operation->id = PSA_CRYPTO_DRIVER_CC3XX_ID;
```

Comment:
```suggestion
        operation->id = PSA_CRYPTO_CC3XX_DRIVER_ID;
```

The original one is more in line with the Mbed TLS convention. Besides, the second macro is undeclared at this point.

Reply:
Not there anymore.